dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22MiscellaneousTests.java

```diff
@@ -516,4 +519,49 @@ public void testCustomValueBindingTag() throws Exception {
             }
         }
     }
+
+    /*
+     * https://github.com/OpenLiberty/open-liberty/issues/20950
+     * https://issues.apache.org/jira/browse/MYFACES-4433
+     * 
+     *  Verifies there is no leak when ViewScope Beans are used.
+     *  There was previously a memory leak with the ViewScopeBeanHolder's storageMap. See issues above.
+     */
+    @Test
+    public void testMyFaces4433() throws Exception {
+        try (WebClient webClient = new WebClient()) {
+
+            URL url = JSFUtils.createHttpUrl(jsf22MiscellaneousServer, "ViewScopeLeak", "index.xhtml");
+
+            String size1 = "1";
+            String size2 = "2";
+
+            // Runs twice. Goes to index.xhtml to create a view new. Goes to new page (invalidate) to end the view.
+            // invalidate.xhtml will list size of WELD_S#0 via SessionSizeHelper. 
+            // The size should be the same for runs.
+            // WELD_S#0 is the attribute where the ViewScope related storage is saved via CDI. 
```

Comment:
1) Should it read: `create a new view`?
2) The size should be the same for **both** runs?